example/config.yml

```
@@ -1,7 +1,7 @@
 receivers:
   kubearmor_receiver:
-    endpoint: ${env:KUBEARMOR_SERVICE?default=:32767}
-    logfilter: ${env:LOGFILTER?default=all}
+    endpoint: :32767
+    logfilter: all
```

What I actually meant was removing the `default` and only having something like:
```suggestion
    endpoint: ${env:KUBEARMOR_SERVICE}
    logfilter: ${env:LOGFILTER}
```
This would allow users to override these values without specifying an entirely new config file. Something like:
```
docker run -e KUBEARMOR_SERVICE=kubearmor:1234 -e LOGFILTER=policy kubearmor/otel-receiver
```
The default values are already set first by the config file present in container and then also by the collector internally, I believe.